### PR TITLE
Feature EE-835: Added Is Bundle check in Module Management window.

### PR DESCRIPTION
**Pull request by Matias-Bernal**

### CODE REVIEW

**Review by RomanMagnoli — APPROVED** [review]

File: src-db/database/model/tables/ETDEP_PACKAGE.xml [hunk 1]
```diff
@@ -49,13 +49,18 @@
         <default/>
         <onCreateDefault/>
       </column>
+      <column name="BUNDLE" primaryKey="false" required="true" type="CHAR" size="1" autoIncrement="false">
+        <default><![CDATA[N]]></default>
+        <onCreateDefault/>
```
Comment (RubenEtendo) [inline src-db/database/model/tables/ETDEP_PACKAGE.xml]: For tables already populated with data, the creation of a new mandatory column should include a value set for it in the `onCreateDefault` property

File: src-db/database/sourcedata/AD_ELEMENT.xml [hunk 2]
```diff
@@ -24,6 +24,18 @@
 <!--0D7EF88E550C4379B886046521156D5C-->  <ISGLOSSARY><![CDATA[N]]></ISGLOSSARY>
 <!--0D7EF88E550C4379B886046521156D5C--></AD_ELEMENT>
 
+<!--1926046ADA624DC3A0583378CF753147--><AD_ELEMENT>
+<!--1926046ADA624DC3A0583378CF753147-->  <AD_ELEMENT_ID><![CDATA[1926046ADA624DC3A0583378CF753147]]></AD_ELEMENT_ID>
+<!--1926046ADA624DC3A0583378CF753147-->  <AD_CLIENT_ID><![CDATA[0]]></AD_CLIENT_ID>
+<!--1926046ADA624DC3A0583378CF753147-->  <AD_ORG_ID><![CDATA[0]]></AD_ORG_ID>
+<!--1926046ADA624DC3A0583378CF753147-->  <ISACTIVE><![CDATA[Y]]></ISACTIVE>
+<!--1926046ADA624DC3A0583378CF753147-->  <COLUMNNAME><![CDATA[Bundle]]></COLUMNNAME>
+<!--1926046ADA624DC3A0583378CF753147-->  <NAME><![CDATA[Is Bundle]]></NAME>
```
Comment (RubenEtendo) [inline src-db/database/sourcedata/AD_ELEMENT.xml:33]: From now on, all new Application Dictionary entries need a description **and** help to better understand their motive

**Review by RubenEtendo — CHANGES_REQUESTED** [review]

**Review by sebastianbarrozo — APPROVED** [review]

File: src-db/database/sourcedata/AD_ELEMENT.xml [hunk 3]
```diff
@@ -24,6 +24,20 @@
...
+<!--1926046ADA624DC3A0583378CF753147-->  <PRINTNAME><![CDATA[Is Bundle]]></PRINTNAME>
+<!--1926046ADA624DC3A0583378CF753147-->  <DESCRIPTION><![CDATA[This field identified the modules that are bundles.]]></DESCRIPTION>
```
Comment (RubenEtendo) [inline src-db/database/sourcedata/AD_ELEMENT.xml:35]: This field **identifies** the modules that are bundles.

File: src-db/database/sourcedata/AD_ELEMENT.xml [hunk 4]
```diff
@@ -24,6 +24,20 @@
...
+<!--1926046ADA624DC3A0583378CF753147-->  <HELP><![CDATA[If this record has Is Bundle check marked then this module is a bundle.]]></HELP>
```
Comment (RubenEtendo) [inline src-db/database/sourcedata/AD_ELEMENT.xml:36]: If this record has **the 'Is Bundle'** check marked, then this module is a bundle

**Review by RubenEtendo — APPROVED** [review]

**Pull request merged**